packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -982,17 +1002,24 @@ class _CupertinoDatePickerDateTimeState extends State<CupertinoDatePicker> {
 
     for (int i = 0; i < columnWidths.length; i++) {
       double offAxisFraction = 0.0;
-      if (i == 0)
+      Widget magnifier;
+      if (i == 0) {
         offAxisFraction = -_kMaximumOffAxisFraction * textDirectionFactor;
-      else if (i >= 2 || columnWidths.length == 2)
+        magnifier = _buildMagnifier(context, _leftEdgeInsets, _leftBorderRadius);
+      } else
```

Comment:
nit: `else if` should be on the same line?

Reply:
Good point, I'm not very good at English, thanks for the advice.

Reply:
That's my formatting omission, thank you.